### PR TITLE
Refine logs when search with unsupported metric type

### PR DESCRIPTION
**Pull request by longjiquan**
Signed-off-by: dragondriver <jiquan.long@zilliz.com>
issue: #12148

### CODE REVIEW

**Comment by scsven** [comment]
@DragonDriver Please add a "kind" label for this pull request

File: internal/core/src/segcore/Utils.h [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions and limitations under the License
+
+#include <stdlib.h>
+#include <string>
+#include <exception>
+#include <stdexcept>
+#include "index/thirdparty/faiss/MetricType.h"
+
+namespace milvus {
```
Comment (cydrain) [inline internal/core/src/segcore/Utils.h:18]: namespace milvus::segcore {

}

**Pull request merged**